packages/node_modules/@webex/test-users/test/integration/spec/whistler.js

```diff
@@ -0,0 +1,70 @@
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import {createWhistlerTestUser, removeWhistlerTestUser} from '@webex/test-users';
+import {nodeOnly} from '@webex/test-helper-mocha';
+
+const {assert} = chai;
+
+chai.use(chaiAsPromised);
+
+assert.hasAccessToken = (user) => {
+  assert.isDefined(user.token, 'user.token is defined');
+};
+
+assert.isTestUser = (user) => {
+  assert.isDefined(user, 'user is defined');
+  assert.isDefined(user.displayName, 'user.displayName is defined');
+  assert.isDefined(user.emailAddress, 'user.emailAddress is defined');
+  assert.isDefined(user.password, 'user.password is defined');
+  assert.isDefined(user.reservationUrl, 'user.reservationUrl is defined');
+};
+
+nodeOnly(describe)('test-users', () => {
+  function prune(user) {
+    return {
+      password: user.password,
+      emailAddress: user.emailAddress,
+      displayName: user.displayName,
+      token: user.token,
+      reservationUrl: user.reservationUrl
+    };
+  }
+
+  describe('createWhistlerTestUser()', () => {
+    it('creates a test user', () => createWhistlerTestUser()
+      .then((u) => {
+        assert.isTestUser(u);
+        assert.hasAccessToken(u);
+      }));
+  });
+
+  describe('removeWhistlerTestUser()', () => {
+    it('removes the specified test user', () => createWhistlerTestUser()
+      .then(async (u) => {
+        const res = await removeWhistlerTestUser(u);
+
+        assert.equal(res.statusCode, 204);
+      }));
+
+    it('removes a test user if no access token is available', () => createWhistlerTestUser()
+      .then(prune)
+      .then(async (u) => {
+        const res = await removeWhistlerTestUser(u);
+
+        assert.equal(res.statusCode, 204);
+      }));
+
+    it('removes a test user with an access token set to undefined', () => createWhistlerTestUser()
```

Comment:
If there is no access token, how do you know what user to remove? or what gets removed?